pkgs/development/python-modules/geomag/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "geomag";
+  version = "0.9.2015";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-+j91RL7KLdcGwweNnVjLNXNNAN4lkScKM1IWu4ujENU=";
+    extension = "zip";
+  };
+
+  doCheck = false;
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "geomag" ];
+
+  meta = {
+    description = "Calculates magnetic variation/declination for any latitude/longitude/altitude,
+for any date.";
```

```suggestion
    description = "Calculates magnetic variation/declination for any latitude/longitude/altitude,
for any date";
```

From https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes 

- Not end with a period (or any punctuation for that matter).
  - Not end with a period (or any punctuation for that matter).